concepts/0003-protocols/uris.md

```diff
@@ -54,39 +53,35 @@ to look up the named and versioned protocol. Optionally and preferably, the
 full URI may produce a page of documentation about the specific message type,
 with no human mediation involved.
 
-### PIURI
+#### PIURI
 
 A shorter URI that follows the same conventions but lacks the
 `message-type-name` portion is called a __protocol identifier URI__
 (PIURI).
- 
- 
+
 ![PIURI structure](piuri-structure.png)
- 
+
 ```ABNF
-protocol-identifier-uri  = doc-uri delim protocol-name 
+protocol-identifier-uri  = doc-uri delim protocol-name
     "/" semver
 ```
 
 Its loose matcher regex is:
 
+```regex
     (.*?)([a-z0-9._-]+)/(\d[^/]*)/?$
-    
-Some examples of valid MTURIs and PIURIs include:
-
-* `http://example.com/protocols?which=lets_do_lunch/1.0/` (PIURI with fully automated lookup of protocol docs)
-* `http://example.com/message_types?which=lets_do_lunch/1.0/proposal` (MTURI)
-* `https://github.com/hyperledger/aries-rfcs/tree/18c4f82:trust_ping/1.0/ping`
-   (MTURI). Note that this URI returns a 404 error if followed directly--but
-   per rules described above, the developer should browse to the doc root
-   ([https://github.com/hyperledger/aries-rfcs/tree/18c4f82](
-   https://github.com/hyperledger/aries-rfcs/tree/18c4f82
-   )) and look for documentation on the `trust_ping/1.0` protocol.
-* `did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/trust_ping/1.0/ping` (MTURI) This
-   uses a DID reference to look up an endpoint named `spec` that serves
-   information about protocols. (The exact syntax of DID references--URIs
-   where the DID functions like a domain name, and additional info is
-   fetched from a DID Doc in much the same way IP address and hostname
-   definitions are fetched from a DNS record--is still being finalized.
-   See the latest [DID Spec](https://w3c-ccg.github.io/did-spec/) for details.)
+```
```

Comment:
I'd prefer all of the old examples and none of the new ones. The new ones assume Aries conventions. This isn't a doc about how core protocols are written in Aries; it's a doc about how any protocols can be written, with Aries just being the place for a core subset.

Reply:
I would like at least some examples of how the URIs are actually used, hence the inclusion of the core ones here and the references.

I didn't understand why the examples removed were there.  We don't want non-core implementers to use those formats.  Where do the query parameters come from?  Are they supported?  By what?  What is the commit reference in the `trust-ping` example?  What is the point of having a URL that returns a 404?

I could not come up with a description of why they were there.  Open to adding but saying the are "valid" seems a stretch.

Reply:
>We don't want non-core implementers to use those formats.

This sentence tells me that we have a fundamentally different assumption about purpose. We don't write Aries RFCs to guide core implementers only; we write them to guide all implementers, everywhere. Some implementers will be core implementers, and they should use the patterns we chosen for Aries core stuff. But some implementers won't be core implementers. They need to know how to follow guidelines even if their context is different. This RFC is guidance for all protocol implementers, not just core implementers.

Reply:
Reverted.  Still haven't a clue what to make of the examples.